Formula/lastpass-cli.rb

```diff
@@ -21,10 +21,12 @@ class LastpassCli < Formula
   depends_on "docbook-xsl" => :build
   depends_on "pkg-config" => :build
   # Avoid crashes on Mojave's version of libcurl (https://github.com/lastpass/lastpass-cli/issues/427)
-  depends_on "curl" if MacOS.version >= :mojave
+  depends_on "curl"
```

Comment:
We should probably keep the `if` statement; there's still a High Sierra bottle users can install without needing to also install `curl`.

Reply:
In that case what's the best way to make this also depends on `curl` on Linux?

Reply:
I guess keep this line but also add a `uses_from_macos "curl"` later on, since that's a no-op on Mac anyway